comictalker/__init__.py

```diff
@@ -30,7 +32,15 @@ def get_talkers(version: str, cache: pathlib.Path) -> dict[str, ComicTalker]:
             if obj.id != talker.name:
                 logger.error("Talker ID must be the same as the entry point name")
                 continue
-            talkers[talker.name] = obj
+            try:
+                if version in SpecifierSet(obj.ct_req_spec, prereleases=True):
+                    talkers[talker.name] = obj
+                else:
+                    logger.error(
+                        f"CT required version not met for talker: {talker.name} with specifier: {obj.ct_req_spec}"
+                    )
+            except InvalidSpecifier:
+                logger.error(f"Invalid specifier for talker: {talker.name} - specifier: {obj.ct_req_spec}")
```

Comment:
How about we just make it minimum version instead of using the full specification

Reply:
My thinking was in case there are breaking changes a max version could be set.

Reply:
My rationale is that if there is a breaking change they will have to make a code change to exclude it, and instead they should make the needed code changes